_posts/blog/2018-11-12/2018-11-12-how-will-the-bch-hard-fork-on-nov-15-affect-openbazaar.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "How Will The <span class='post-title-extra'>BCH Hard Fork</span> Affect OpenBazaar?"
+layout: post
+date: '2018-11-12 06:30:00 -0300'
+social_title: 'How Will The Bitcoin Cash (BCH) Hard Fork on November 15th Affect OpenBazaar?'
+social_description: 'Here is what you need to know about the Bitcoin Cash hard fork on November 15th as it relates to Openbazaar.'
+---
+
+#### What’s Happening with Bitcoin Cash?
+
+There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade due to complete at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.
+
+#### How Will The Bitcoin Cash Upgrade Affect My OpenBazaar Funds?
+
+Users of OpenBazaar 2.2.5 and prior who have an active BCH wallet are using a local SPV-based wallet. [SPV wallets](https://bitcoin.org/en/glossary/simplified-payment-verification) are designed to follow [Nakamoto Consensus](https://blockonomi.com/nakamoto-consensus/), which means the fork that has the most cumulative proof-of-work according to the existing Bitcoin Cash difficulty rules. 
```

Comment:
I would definitely NOT refer to this a Nakamoto Consensus. Nakamoto Consensus is a way of choosing between two valid chains, not between two chains with different rules. 

It's somewhat of a flaw in the design of SPV wallets that they can't tell when the chain has invalid consensus rules. I have a design for a new SPV wallet that can do so, but it's not deployed. 

So for now I would just say, due to the design limitations the SPV wallet will follow the longest chain of the peers it's able to connect to. That may be the Bitcoin ABC chain or the BItcoin SV chain as it is rather random which peers you connect to. 

Users who want to force the software to follow a specific side of the fork should set an IP address of a known peer on the side of the chain they want to follow as the "trustedPeer" in the config file.